src/screens/TransactionDetailsScreen.tsx

```diff
@@ -234,7 +234,7 @@ class TransactionDetailsScreen extends Component<Props> {
           <StyledText
             title={i18n.transactions.details.addToAddressBook}
             onPress={() => {
-              this.addToAddressBook(fromValue);
+              this.addToAddressBook(fromValue.split(',')[0]);
```

Comment:
this is used also in line 231, please assign to const

Reply:
done :)